kpops/components/base_components/kubernetes_app.py

```diff
@@ -19,7 +19,14 @@
 
 
 class KubernetesAppValues(CamelCaseConfigModel, DescConfigModel):
-    """Settings specific to Kubernetes apps."""
+    """Settings specific to Kubernetes apps.
+
+    :param image_tag: The image tag of the Kubernetes app.
+    """
+
+    image_tag: str = Field(
```

Comment:
is this standard described somewhere by Kubernetes docs? I was under the impression that it's more of a best practice thing originating in our Helm charts

Reply:
I pushed a new definition

Reply:
it doesn't answer my question though. is this even standardized by Kubernetes?

Reply:
Regarding the helm charts, we can use the command `show values` to fetch the values from the `values.yaml` file as the default.
https://helm.sh/docs/helm/helm_show_values/#helm-show-values

Reply:
I think we're talking at cross purposes 😄 what I am curious about is if the `imageTag` attribute is standardized by Kubernetes?

Reply:
it is actually set as the `image:tag` in the Deployment, Pod, StatefulSet, etc.
https://kubernetes.io/docs/concepts/containers/images/#:~:text=If%20you%20don%27t%20specify%20a%20tag%2C%20Kubernetes%20assumes%20you%20mean%20the%20tag%20latest.

Reply:
I would rather have it in `KafkaAppValues`.

Reply:
but it's not related to `KafkaApp` 🤷🏻‍♂️